docs/tutorials/tokens/erc1155.md

```diff
@@ -164,3 +164,64 @@ Other parameters:
 - You must specify a `pool` name if you've created more than one pool
 - You may specify a `key` understood by the connector (i.e. an Ethereum address) if you'd like to use a non-default signing identity
 - You may specify `from` if you'd like to burn tokens from a specific identity (default is the same as `key`)
+
+## Token approvals
+You can also approve other wallets to transfer tokens on your behalf with the `/approvals` API. The important fields in a token approval API request are as follows:
+
+- `approved`: Sets whether another account is allowed to transfer tokens out of this wallet or not. If not specified, will default to `false`.
+- `operator`: The other account that is allowed to transfer tokens out of the wallet specified in the `key` field
+- `config.amount`: The amount of tokens the other account is allowed to transfer. If `0` or not set, the approval is valid for any number. Required for `fungible` token pools.
+- `config.tokenIndex`: The specific token index within the pool that the operator is allowed to transfer. If `0` or not set, the approval is valid for any token. Required for `nonfungible` token pools.
+- `key`: The wallet address for the approval. If not set, it defaults to the address of the FireFly node submitting the transaction
+
+Here is an example request that would let the signing account `0x634ee8c7d0894d086c7af1fc8514736aed251528` transfer up to 10×10<sup>18</sup> (`10000000000000000000`) tokens from my wallet
+
+#### Request
+`POST` `http://127.0.0.1:5000/api/v1/namespaces/default/tokens/approvals`
+```json
+{
+  "approved": true,
+  "operator": "0x634ee8c7d0894d086c7af1fc8514736aed251528",
+  "config": {
+      "amount": "10000000000000000000"
+  }
+}
+```
+
+#### Response
+```json
+{
+    "localId": "46fef50a-cf93-4f92-acf8-fae161b37362",
+    "pool": "e1477ed5-7282-48e5-ad9d-1612296bb29d",
+    "connector": "erc20_erc721",
+    "key": "0x14ddd36a0c2f747130915bf5214061b1e4bec74c",
+    "operator": "0x634ee8c7d0894d086c7af1fc8514736aed251528",
+    "approved": true,
+    "tx": {
+        "type": "token_approval",
+        "id": "00faa011-f42c-403d-a047-2df7318967cd"
+    },
+    "config": {
+        "amount": "10000000000000000000"
+    }
+}
+```
+
+#### Response
+```json
+{
+    "localId": "46fef50a-cf93-4f92-acf8-fae161b37362",
+    "pool": "e1477ed5-7282-48e5-ad9d-1612296bb29d",
+    "connector": "erc1155",
+    "key": "0x14ddd36a0c2f747130915bf5214061b1e4bec74c",
+    "operator": "0x634ee8c7d0894d086c7af1fc8514736aed251528",
+    "approved": true,
+    "tx": {
+        "type": "token_approval",
+        "id": "00faa011-f42c-403d-a047-2df7318967cd"
+    },
+    "config": {
+        "amount": "10000000000000000000"
+    }
+}
+```burning a non-fungible token.
```

Comment:
Some stray response JSON and text down here?

Reply:
yep... 🤦‍♂️